compiler/src/main/kotlin/io/spine/protodata/renderer/InsertionPoint.kt

```diff
@@ -49,7 +64,84 @@ public interface InsertionPoint {
      * @return the line number at which the insertion point should be added.
      * @see LineNumber
      */
-    public fun locate(lines: List<String>): LineNumber
+    @Deprecated("Use locate(Text) instead.")
+    public fun locate(lines: List<String>): LineNumber {
+        val coords = locate(text(lines))
+        return when (coords.specCase) {
+            WHOLE_LINE -> LineNumber.at(coords.wholeLine)
+            INLINE -> {
+                loggerFor(InsertionPoint::class.java)
+                    .atWarning()
+                    .withStackTrace(FULL)
+                    .log("`locate(List<String>)` does not support inline insertion. " +
+                            "A whole line insertion will be generated.")
+                LineNumber.at(coords.inline.line)
+            }
+            END_OF_FILE -> LineNumber.endOfFile()
+            NOT_IN_FILE -> LineNumber.notInFile()
```

Comment:
How can we have a “line number not in file”? It's just “not in file”. It can be anywhere around the file. Lines are just not applicable in this case.

Reply:
`LineNumber` is the old type that is being deprecated and replaced by `TextCoordinates`, so I won't touch it now.